wolfssl/openssl/evp.h

```diff
@@ -547,7 +547,11 @@ WOLFSSL_API int wolfSSL_EVP_PKEY_keygen_init(WOLFSSL_EVP_PKEY_CTX *ctx);
 WOLFSSL_API int wolfSSL_EVP_PKEY_keygen(WOLFSSL_EVP_PKEY_CTX *ctx,
   WOLFSSL_EVP_PKEY **ppkey);
 WOLFSSL_API int wolfSSL_EVP_PKEY_bits(const WOLFSSL_EVP_PKEY *pkey);
+#ifdef WOLFSSL_QT
```

Comment:
Switch on OPENSSL_VERSION_NUMBER instead of WOLFSSL_QT

Reply:
Guard by OPENSSL_VERSION_NUMBER instead of WOLFSSL_QT